VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/private-consultation-room-controls/private-consultation-room-controls.component.html

```diff
@@ -91,6 +91,11 @@
                             : 'private-consultation-room-controls.hand-raise-toggle-false') | translate
                             }}</span></span>
                 </div>
+                <div *ngIf='!isPrivateConsultation && isHost' id='leave-hearing' class='icon-button-dark hide-for-medium'
+                    data-module='govuk-button' (click)='displayLeaveHearingPopup = !displayLeaveHearingPopup'>
```

Comment:
I think this can just be true and not a toggle since the button shouldn't actually be clickable when the modal is visible.